sklearn/utils/fast_dict.pxd

```diff
@@ -8,6 +8,7 @@ integers, and values float.
 from libcpp.map cimport map as cpp_map
 
 # Import the C-level symbols of numpy
+import numpy as np
```

Comment:
this is fine, except for the comment preceding it is now inappropriate. Please move it above the comment.

Reply:
@jnothman Thank you. I have moved the import statement up before the comment.